contrib/labstack/echo.v4/option.go

```diff
@@ -33,11 +34,15 @@ type Option func(*config)
 // IgnoreRequestFunc determines if tracing will be skipped for a request.
 type IgnoreRequestFunc func(c echo.Context) bool
 
+// CustomErrorFunc attempts to translate a non-standard echo error to the required format
+type CustomErrorFunc func(err error) *echo.HTTPError
+
 func defaults(cfg *config) {
 	cfg.serviceName = namingschema.NewDefaultServiceName(defaultServiceName).GetName()
 	cfg.analyticsRate = math.NaN()
 	cfg.isStatusError = isServerError
 	cfg.headerTags = globalconfig.HeaderTagMap()
+	cfg.customErrorFunc = func(err error) *echo.HTTPError { return nil }
```

Comment:
Also we should document in the comments here that by default we do no translation which is why we're setting the value to nil.